### PR TITLE
[#14] Set up fatal warnings compilation flag

### PR DESCRIPTION
**Pull request by christobill**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -104,7 +105,7 @@ val flagsFor11 = Seq(
 val flagsFor12 = Seq(
   "-Xlint:_",
   "-Ywarn-infer-any",
-  "-opt:l:project"
+  "-opt:l:inline"
```
Comment (mdedetrich) [inline build.sbt]: This should be `"-opt-inline-from:<sources>"`. See https://github.com/mdedetrich/scalajson/issues/43

**Review by mdedetrich — CHANGES_REQUESTED** [review]
Great work, just using the wrong optimization setting.

**Comment by christobill** [comment]
Thanks for the feedback! I updated the MR

**Review by mdedetrich — APPROVED** [review]

**Comment by mdedetrich** [comment]
@christobill Do you need a new version to be released any time soon?

**Comment by christobill** [comment]
@mdedetrich thanks! I will try to propose (and hopefully merge) some MR before a new release :)

**Pull request merged**